experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -77,35 +77,17 @@ export const makeFieldEditGenerator = (
 		// generate edit for that specific tree
 		const { fieldPath, fieldKey, count } = getExistingFieldPath(tree, state.random);
 		assert(fieldPath.parent !== undefined);
-
+		const opWeightRatio =
+			passedOpWeights.insert / (passedOpWeights.delete + passedOpWeights.insert);
+		const opType = count === 0 || state.random.bool(opWeightRatio) ? "insert" : "delete";
```

Comment:
doesn't this mean we generate inserts sometimes when the weight is 0? that seems weird. Maybe it's worth taking a step back and thinking about how this area of the generators is factored.